### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by thakkarparth007** [fastai/nbdev#316]
I was trying nbdev today (and it seems like a great idea!). It works as expected, except for some of the make commands. E.g., I was running this from powershell (could've avoided the issue by running it from WSL), and running `make docs_serve` failed randomly because `make docs` calls `touch docs`, and `touch` isn't available on powershell. Similarly, `make clean` uses `rm -rf dist` which doesn't work in powershell.

Editing the makefiles isn't much of an effort, but doing it for every project might become a little painful. Making this work on Windows out of the box would be useful.

**Comment by nglillywhite** [comment]
Hi @thakkarparth007, as a stop gap you can install touch-cli via npm and the command will work, I ran into this problem as well and touch-cli solved it for me. I think jeremy has already changed the title to reflect windows support for the make file.

if you haven't got npm, you can go here: https://www.npmjs.com/get-npm

after that is installed, the command is 'npm install touch-cli -g'

See this stackoverflow of someone else with the same issue: https://stackoverflow.com/questions/36126269/touch-is-not-recognized-as-an-internal-or-external-command-operable-program-o/36126799

and https://www.npmjs.com/package/touch-cli for the actual package.

Hope that helps in the interim of there being an out the box windows makefile.

**Comment by thakkarparth007** [comment]
Thanks for the pointer. In my case I'd modified the Makefile to suit my specific case, but for future projects this would be a quick workaround.

**Comment by HDavidEspinosa** [comment]
I've been trying to follow the tutorial for nbdev using conda, Windows 10 terminal, and powershell. However, for some reason, I haven't been able to run the nbdev_* commands. I have tried nbdev_install_git_hooks and nbdev_build_lib, as per the instructions. But, as soon as I hit return on the terminal it just asks me for a program to open the file with.

I tried editing the #! line at the beginning to direct the files to gits bin/sh... but it didn't work. I then executed the files as if they were python files. nbdev_install_git_hooks worked, but the trick failed with nbdev_build_lib. 

I really want to try this library, but I also want to avoid having to install WSL and Linux. Can anyone see what I may be doing wrong?

Thanks
